src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -289,6 +290,9 @@ def add_temporal_outcome(
             fallback (float): What to do if no value within the lookahead.
             outcome_df_values_col_name (str): Column name for the outcome values in outcome_df, e.g. whether a patient has t2d or not at the timestamp. Defaults to "value".
             new_col_name (str): Name to use for new col. Automatically generated as '{new_col_name}_within_{lookahead_days}_days'.
+            is_fallback_prop_warning_threshold (float, optional): Triggers a ValueError if proportion of
+                prediction_times that receive fallback is larger than threshold.
+                Indicates unlikely to be a learnable feature. Defaults to 0.9.
```

Comment:
Def. lower it. is 50% too much?

Reply:
Oh, I'm not sure "definitely lower" is the right move here. We'll have many features where lack of information is information.

E.g. if there is no record of a patient receiving a medication, that means that they haven't received it, but it still triggers the fallback.

Which variables would we expect 0.9 to be too high, but 0.5 to be appropriate? :-)